Makefile

```diff
@@ -187,7 +187,7 @@ discover: $(BUILD_DIR)/bin/discover
 token: GO_LDFLAGS=-X $(pkgmap.$(@F))/metadata.Version=$(PROJECT_VERSION)
 token: $(BUILD_DIR)/bin/token
 
-blkvault-repo-docker: $(BUILD_DIR)/image/blkvault-repo/$(DUMMY)
+blkvault-repo-docker: $(BUILD_DIR)/images/blkvault-repo/$(DUMMY)
```

Comment:
I would like to remove all text containing "vault" and replace it with "archiver"

Reply:
Rename all 'vault' to 'archiver', including [fabric-block-archiving-testenv repository](https://github.com/nekia/fabric-block-archiving-testenv).